Add to README.md instructions on how to run UI tests (#888)
anitab-org/mentorship-android: README.md

Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-android/wiki/Commit-Message-Style-Guide) while sending PRs.

## Running the unit tests

> To run the existing unit tests follow the steps given below:-

Reviewer: ```suggestion
To run the existing unit tests follow the steps given below:
```

* Connect your Android device or open the emulator.

> * On the terminal type ./gradlew clean build connectedAndroidTest --stacktrace

Reviewer: ```suggestion
* On the terminal type: `./gradlew clean build connectedAndroidTest --stacktrace`
```

> * If the above command is not working try using the command gradlew clean build connectedAndroidTest --stacktrace

Reviewer: ```suggestion
* If the above command is not working try using the command: `gradlew clean build connectedAndroidTest --stacktrace`
```

* Wait until all the tests are completed.
* You will get a report generated with a test summary.

> * See the complete report ../mentorship-android/app/build/reports/androidTests/connected/index.html

Reviewer: ```suggestion
* See the complete report `./mentorship-android/app/build/reports/androidTests/connected/index.html`
```

> * The report ../mentorship-android/app/build/reports/androidTests/connected/index.html will show all the testsuites that have passed as well as failed.

Reviewer: ```suggestion
* The report `./mentorship-android/app/build/reports/androidTests/connected/index.html` will show all the testsuites that have passed as well as failed.
```

* To get a more detailed explanation about the tests of a particular testsuite visit respective html file of the testsuite.

>    For eg:-org.systers.mentorship.LoginActivityTest.html will contain the results of all the tests run under that particular testsuite.

Reviewer: ```suggestion
   For e.g.: `org.systers.mentorship.LoginActivityTest.html` will contain the results of all the tests run under that particular testsuite.
```

> * You can find the respective html files of the different testsuites under the heading classes in index.html report .

Reviewer: ```suggestion
* You can find the respective html files of the different testsuites under the heading classes in `index.html` report .
```
